dlc/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelfTest.scala

```diff
@@ -93,6 +93,76 @@ class BinaryOutcomeDLCWithSelfTest extends BitcoinSAsyncTest {
     }
   }
 
+  def closeTogether(num1: Double, num2: Double): Boolean = {
+    Math.abs(num1 - num2) < 1.0e-6
```

Comment:
What is `1.0e-6` in terms of sats? Does this mean our remainder here would be theoretically less than 1 satoshi?

Reply:
These are unit-less numbers, not satoshis. We're just dealing with comparing proportions and there will be rounding error and also remainders (off by a bit because of discrete stuff) so I figured I'd just assert that things are close enough.